.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Description
+[Describe what this change achieves]
+ 
+### Issues Resolved
+[List any issues this PR will resolve]
+ 
+### Check List
```

Comment:
Do we need few more checks?
For example - 
1] Check in  both light and dark mode and verify change is visible.
2] Documentation
3] Check in different browsers.
etc.

Reply:
I would love to have a long list but it would be annoying. Following in the footsteps of OSD, this short list should be enough, unless you feel strongly about something.

Reply:
Will revisit this in the next round